app/build.gradle

```diff
@@ -34,7 +47,12 @@ dependencies {
     compile 'com.google.android.gms:play-services:7.5.0'
     compile 'com.parse.bolts:bolts-android:1.+'
     compile 'com.github.navasmdc:MaterialDesign:1.+@aar'
-    compile 'com.nineoldandroids:library:2.4.0'
-    compile 'com.google.http-client:google-http-client-android:1.18.0-rc'
-    compile 'com.google.http-client:google-http-client-jackson:1.18.0-rc'
-}
\ No newline at end of file
+    compile ('com.nineoldandroids:library:2.4.0')
+    compile ('com.google.http-client:google-http-client-android:1.18.0-rc'){
```

Comment:
space after the close parenthesis 

Example: ...1.18.0-rc') { instead of ...1.18.0-rc'){